### PR TITLE
Add degrees to accelerometer for X and Y axis in sensor tab

### LINKED ISSUE #1620: Add Degree field to ACC sensor page
Would be nice to add accelerometer degrees to the sensor page (next to the G's numbers), like -90 to 0 to +90 degrees. Also add a calibrate ACC button on this page (like the setup tab has). TIA!
BFC ACC Degree wanted

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes #1620
Screenshot from 2020-10-06 11-56-53

### CODE REVIEW

File: src/js/tabs/sensors.js [hunk 1]
```diff
@@ -379,9 +379,15 @@ TABS.sensors.initialize = function (callback) {
...
-                    raw_data_text_ements.x[1].text(FC.SENSOR_DATA.accelerometer[0].toFixed(2));
-                    raw_data_text_ements.y[1].text(FC.SENSOR_DATA.accelerometer[1].toFixed(2));
-                    raw_data_text_ements.z[1].text(FC.SENSOR_DATA.accelerometer[2].toFixed(2));
+                    const x = FC.SENSOR_DATA.accelerometer[0].toFixed(2);
+                    const y = FC.SENSOR_DATA.accelerometer[1].toFixed(2);
+                    const z = FC.SENSOR_DATA.accelerometer[2].toFixed(2);
+                    const pi = Math.PI;
+                    const rollACC = Math.round(Math.atan(y / (Math.sqrt(Math.pow(x, 2)) + (Math.pow(z, 2)))) * (180 / pi));
+                    const pitchACC = Math.round(Math.atan(x / (Math.sqrt(Math.pow(y, 2)) + (Math.pow(z, 2)))) * (180 / pi));
+                    raw_data_text_ements.x[1].text(`${x} ( ${rollACC} )`);
```
Comment (McGiverGim) [inline src/js/tabs/sensors.js:388]: As a suggestion, an screen capture will help to see the final result and help with the review. 

Without seeing the final result I suppose is better to remove the spaces with the parenthesis:
```suggestion
                    raw_data_text_ements.x[1].text(`${x} (${rollACC})`);
```

If we have enough space, that I doubt, we can directly add the units:
```suggestion
                    raw_data_text_ements.x[1].text(`${x}g (${rollACC} deg)`);
```
but we will need to move the string to the messages file.

What do you think?

Comment (haslinghuis) [reply src/js/tabs/sensors.js:388]: Updated. The spaces are for visual separating g-force and degrees for roll (X) and pitch (Y) access.

Comment (McGiverGim) [reply src/js/tabs/sensors.js:388]: I was talking about the spaces _inside_ the parenthesis, not between g-forge and degrees. My first suggestion, final result removing it:
image

Comment (haslinghuis) [reply src/js/tabs/sensors.js]: Fixed!

**Review by McGiverGim — APPROVED** [review]
Thanks!!!

**Review by mikeller — APPROVED** [review]

**Pull request merged**